source/reference/mongosqld.txt

```diff
@@ -718,6 +718,17 @@ These configuration file options are also available as system variables.
 For more information, see
 :ref:`System Variables <system-environment-variables>`.
 
+Load External Configuration Values
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You may use expansion directives in your configuration file to load 
+externally sourced configuration values. Expansion directives help 
```

Comment:
This LGTM, but perhaps remove "help", or rewrite it slightly to something like "Use expansion directives to obscure..." if the directives 100% obscure the info.